R/FeatSelControl.R

```diff
@@ -109,12 +111,14 @@
 NULL
 
 makeFeatSelControl = function(same.resampling.instance, impute.val = NULL, maxit, max.features,
-  tune.threshold = FALSE, tune.threshold.args = list(), log.fun = NULL, ..., cl) {
+  tune.threshold = FALSE, tune.threshold.args = list(), log.fun = "default", ..., cl) {
```

Comment:
Would be (imho unnecessary) complexity for the user. There would be two more functions on the export list which have only the specific purpose of being fed to `log.fun`.